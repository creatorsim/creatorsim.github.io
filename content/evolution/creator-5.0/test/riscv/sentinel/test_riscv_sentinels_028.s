
#
# ARCOS.INF.UC3M.ES
# BY-NC-SA (https://creativecommons.org/licenses/by-nc-sa/4.0/deed.es)
#


.text

    main: 
            
            li ra, 5
            jal ra, test

            # exit
            li a7, 10
            ecall


    test:
            # crear "stack frame" para ra, fp y una variable local
            
            li ra, 4234

            # return a7
           jr ra
