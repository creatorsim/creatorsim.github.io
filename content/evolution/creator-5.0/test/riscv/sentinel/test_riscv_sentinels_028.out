
cr[PC]:0x108a; ir[x1,ra]:0x108a; keyboard[0x0]:''; display[0x0]:''; 

